infrastructure/aws/ecs.tf

```diff
@@ -1,8 +1,8 @@
 locals {
   record_prefix = terraform.workspace == "prod" ? var.project_name : "${var.project_name}-${terraform.workspace}"
-  host          = "${local.record_prefix}.${var.domain_name}"
+  django_host          = "${local.record_prefix}.${var.domain_name}"
+  api_host          = "${local.record_prefix}-api.${var.domain_name}"
```

Comment:
all this is very uncertain and subject to change pending a spike into how we manage https requests between apps